packages/synapse-react-client/src/lib/containers/markdown/MarkdownSynapse.tsx

```diff
@@ -442,6 +442,12 @@ export default class MarkdownSynapse extends React.Component<
               {children}
             </Typography>
           )
+        case 'a':
+          return (
+            <Link {...props} component={Tag}>
+              {children}
+            </Link>
+          )
```

Comment:
Let MUI handle `a` tags. This doesn't include the `a` button case, because that's handled separately by `MarkdownWidget`.